### DISCUSSION TITLE
Websocket in Chatwoot

### DISCUSSION THREAD

**Discussion by hoaibao256** [chatwoot/chatwoot#5342] [Chatwoot APIs]
Hello,
I have built a project with a message dashboard including all conversations in Chatwoot. Now I want to use WebSocket to make it real-time.

- I have followed that link: https://www.chatwoot.com/docs/product/others/websocket-events?fbclid=IwAR2edN4aMnGajmAZBfCJ_4cIY_Ztn6Yo6NaQDlHtlJqXUddXBWSL6QuyG_w but I got an error "WebSocket connection to 'ws://http//localhost:8000/admin/chatwoot/' failed: ".
- I build a module in perfex crm like that:
![image](https://user-images.githubusercontent.com/83713864/186878737-16038b1a-21a1-4ae5-9ee6-2dc6cc03dc02.png)

**Comment by bernapastorini** [comment]
Hi! According to the documentation, the connection string is 'wss://YOUR_CHATWOOT_URL/cable' and you're trying to connect to 'ws://http//localhost:8000/admin/chatwoot/'.

I hope I have helped you.

**Reply by hoaibao256** [reply]
"wss://app.chatwoot.com/app/accounts/74427" like that ?

**Reply by hoaibao256** [reply]
I have tried "wss://app.chatwoot.com/app/accounts/74427"  and     "ws://localhost:8000/cable"
 but they don't work. Here is my page.
![image](https://user-images.githubusercontent.com/83713864/187332217-dcc621a2-e9ad-4c17-97db-3ef9a03625fb.png)
